app/assets/stylesheets/partials/stamps.scss

```diff
@@ -1,4 +1,36 @@
+/*Global:*/
```

Comment:
ist das wirklich global? Also auf *alles* bezogen? Wenn ja -> schieb es in `global.scss`. Sonst den Comment umbenennen